eng/CredScanSuppression.json

```diff
@@ -50,11 +51,13 @@
         },
         {
             "file":[
+                "sdk/core/Azure.Core/tests/PemReaderTests.Keys.cs",
```

Comment:
I'm a bit worried it would serve as a bad example for people to add their entire cs files to ignore list.

Reply:
They could've before.

I do include a comment that only keys should be in there (there is one test but because it needs a variable for line endings). Making these separate files and having to templatize some of them is an unnecessary burden.

Reply:
I definitely don't want entire cs files blindly added but if the cs file only contains test secrets and it is clear that is what the file is for I'm OK with this approach.

Reply:
I remember seeing an inline ignore syntax in CredScan similar to what cspell has. Should we use it instead?

Reply:
There is some support for [inline suppressions](https://strikecommunity.azurewebsites.net/articles/4124/inline-suppression-usage-examples.html) however I've been trying to avoid them because they are CredScan specific and there are other secret scanning tools we are also depending on (i.e. github secret scanning) which don't support them and having secrets being isolated to individual files or using a set of placeholders is a little easier to reason about in those contexts.